docs/semgrep-code/remove-duplicates.md

```diff
@@ -5,26 +5,34 @@ title: Remove duplicate findings
 hide_title: true
 description: Learn how to remove duplicate findings and prevent them from displayed in Semgrep AppSec Platform.
 tags:
-  - Semgrep Code
-  - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
 # Remove duplicate findings
 
 Semgrep scans are performed on both mainline (trunk) and non-mainline branches. The scope of the scan can differ depending on if Semgrep is called on a mainline or non-mainline branch.
 
 <dl>
-    <dt>Full scan</dt>
-    <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
-    <dt>Diff-aware scan</dt>
-    <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
+ <dt>Full scan</dt>
+ <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
+ <dt>Diff-aware scan</dt>
+ <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
 </dl>
 
-## Remove duplicate findings using Semgrep AppSec Platform
+Regardless of scan type, Semgrep generates a finding whenever a Semgrep rule matches a piece of code.
 
-Regardless of the scope of a scan, Semgrep correlates findings across branches based on their unique fingerprint, automatically deduplicating findings and making it simpler to triage.
+For each finding identified by Semgrep, Semgrep tags the finding with a unique fingerprint that allows Semgrep to track the finding as it exists in a file over time. Semgrep generates this fingerprint by combining and hashing information retrieved during runtime, including file path, rule name, and pattern, and the metavariables substituted into the pattern.
```

Comment:
I think it's fine to include this information in our docs, my suggestion would be to just include some sort of callout to let users know that this calculation could change and be improved upon any time. 

We'd ideally keep this doc updated when we do, so hopefully that will help with the reliability of this information